### PR TITLE
[FLINK-18737][docs]translate jdbc connector

### PR DESCRIPTION
**Pull request by weizheng92**
## What is the purpose of the change

Translate jdbc document

## Brief change log

Translate jdbc document

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper:no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by weizheng92** [comment]
@flinkbot  This  issue has been assigned to me

File: docs/dev/connectors/jdbc.zh.md [hunk 1]
```diff
@@ -38,10 +38,10 @@ To use it, add the following dependency to your project (along with your JDBC-dr
 </dependency>
 {% endhighlight %}
 
-Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{ site.baseurl}}/dev/project-configuration.html).
+注意连接器目前还 __不是__ 二进制发行版的一部分，如何在集群中运行请参考 [这里] ({{ site.baseurl}}/dev/project-configuration.html)。
```
Comment (XBaith) [inline docs/dev/connectors/jdbc.zh.md:41]:
```suggestion
注意连接器目前还 __不是__ 二进制发行版的一部分，如何在集群中运行请参考 [这里] ({{ site.baseurl}}/zh/dev/project-configuration.html)。
```

File: docs/dev/connectors/jdbc.zh.md [hunk 2]
```diff
@@ -38,10 +38,10 @@ To use it, add the following dependency to your project (along with your JDBC-dr
...
 
-Created JDBC sink provides at-least-once guarantee.
-Effectively exactly-once can be achived using upsert statements or idempotent updates.
+已创建的JDBC Sink能够保证至少一次的语义。
+更有效的精确执行一次可以通过upsert语句或幂等更新实现。
 
 Example usage:
```
Comment (XBaith) [inline docs/dev/connectors/jdbc.zh.md:46]: 这个也可以翻译一下

Comment (weizheng92) [reply docs/dev/connectors/jdbc.zh.md:46]: 感谢回复，我会及时修改

**Comment by weizheng92** [comment]
@Jark Wu  could plz have a review?

**Comment by weizheng92** [comment]
@XBaith Wu could plz have a review again?

File: docs/dev/connectors/jdbc.zh.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ under the License.
 * This will be replaced by the TOC
 {:toc}
 
-This connector provides a sink that writes data to a JDBC database.
+此连接器可以向JDBC数据库写入数据。
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:29]:
```suggestion
此连接器可以向 JDBC 数据库写入数据。
```

File: docs/dev/connectors/jdbc.zh.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ under the License.
...
 
-To use it, add the following dependency to your project (along with your JDBC-driver):
+添加下面的依赖以便使用此连接器（同时添加JDBC驱动）：
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:31]:
```suggestion
添加下面的依赖以便使用此连接器（同时添加 JDBC 驱动）：
```

File: docs/dev/connectors/jdbc.zh.md [hunk 5]
```diff
@@ -38,12 +38,14 @@ To use it, add the following dependency to your project (along with your JDBC-dr
 </dependency>
 {% endhighlight %}
 
-Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{ site.baseurl}}/dev/project-configuration.html).
+注意连接器目前还 __不是__ 二进制发行版的一部分，如何在集群中运行请参考 [这里] ({{ site.baseurl}}/zh/dev/project-configuration.html)。
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:41]: 这里链接有问题

大概是这样的
```
[这里]({%link project-configuration.zh.md %})
```
具体的可以参考 [这里](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/Reminder-Prefer-link-tag-in-documentation-td42362.html)

这个可以修改完成后，在本地执行 `./docs/docker/run.sh` 然后执行 `./build_docs.sh -p` 然后打开 localshot:4000 查看具体的效果

File: docs/dev/connectors/jdbc.zh.md [hunk 6]
```diff
@@ -38,12 +38,14 @@ To use it, add the following dependency to your project (along with your JDBC-dr
...
 
-Created JDBC sink provides at-least-once guarantee.
-Effectively exactly-once can be achieved using upsert statements or idempotent updates.
 
-Example usage:
+已创建的JDBC Sink能够保证至少一次的语义。
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:44]:
```suggestion
已创建的 JDBC Sink 能够保证至少一次的语义。
```

File: docs/dev/connectors/jdbc.zh.md [hunk 7]
```diff
@@ -38,12 +38,14 @@ To use it, add the following dependency to your project (along with your JDBC-dr
...
+更有效的精确执行一次可以通过upsert语句或幂等更新实现。
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:45]:
```suggestion
更有效的精确执行一次可以通过 upsert 语句或幂等更新实现。
```

File: docs/dev/connectors/jdbc.zh.md [hunk 8]
```diff
@@ -64,4 +66,4 @@ env
 env.execute();
 {% endhighlight %}
 
-Please refer to the [API documentation]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/connector/jdbc/JdbcSink.html) for more details.
+更多细节请查看 [API documentation]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/connector/jdbc/JdbcSink.html) 。
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:69]: 这里的链接也建议使用上面的方式进行修改

**Review by klion26 — COMMENTED** [review]
@weizheng92 thanks for your contribution. left some comments. please take a look.

File: docs/dev/connectors/jdbc.zh.md (hunk 5 above)
Comment (weizheng92) [reply docs/dev/connectors/jdbc.zh.md]: 非常感谢您的指导，这对我帮助非常大。

File: docs/dev/connectors/jdbc.zh.md (hunk 8 above)
Comment (weizheng92) [reply docs/dev/connectors/jdbc.zh.md:69]: 非常感谢您的回复，我已经修改，并按您的指导在本地查看了效果。
其中第二个链接为外部链接，请问还需要修改吗。

**Comment by weizheng92** [comment]
@klion26 您好，已经修改完成，有时间再看一下吗，非常感谢。

File: docs/dev/connectors/jdbc.zh.md [hunk 9]
```diff
@@ -38,12 +38,12 @@ To use it, add the following dependency to your project (along with your JDBC-dr
 </dependency>
 {% endhighlight %}
 
-Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{ site.baseurl}}/dev/project-configuration.html).
+注意连接器目前还 __不是__ 二进制发行版的一部分，如何在集群中运行请参考 [这里]({% link dev/project-configuration.zh.md %})。
```
Comment (klion26) [inline docs/dev/connectors/jdbc.zh.md:41]: `注意连接器` -> `注意该连接器` 会好一些吗？
现在的描述读起来有一点点拗口

File: docs/dev/connectors/jdbc.zh.md (hunk 8 above)
Comment (klion26) [reply docs/dev/connectors/jdbc.zh.md:67]: 或许你可以试试 `({% link /api/java/org/apache/flink/connector/jdbc/JdbcSink.html %})`

**Review by klion26 — COMMENTED** [review]
@weizheng92 thanks for the update, left two more minor comments. please take another look.

File: docs/dev/connectors/jdbc.zh.md (hunk 9 above)
Comment (weizheng92) [reply docs/dev/connectors/jdbc.zh.md]: 感谢回复，已经修改。

File: docs/dev/connectors/jdbc.zh.md (hunk 8 above)
Comment (weizheng92) [reply docs/dev/connectors/jdbc.zh.md:67]: 已经尝试这种方式  ({% link /api/java/org/apache/flink/connector/jdbc/JdbcSink.html %})，但是外部链接可能有问题，得到如下错误：
Liquid Exception: Could not find document '/api/java/org/apache/flink/connector/jdbc/JdbcSink.html' in tag 'link'.

**Review by klion26 — APPROVED** [review]
@weizheng92 thanks for the update, LGTM now
ping @wuchong @rmetzger

**Pull request merged**